### PR TITLE
Scroll to Staff Label

### PR DESCRIPTION
**Pull request by ciduarte**
On return to the Edit Course Page from the Staffer Page,
open the collapsible for the associated Course Run and
scroll to the Staff field label.

DISCO-1111

### CODE REVIEW

File: src/components/EditCoursePage/CollapsibleCourseRun.jsx [hunk 1]
```diff
@@ -87,10 +87,54 @@ class CollapsibleCourseRun extends React.Component {
...
+    if (wasReturnedFromStaff && !returnFromStaff) {
+      const element = document.getElementById(`${courseId}.staff.label`);
+      element.scrollIntoView({ behavior: 'smooth', block: 'start', inline: 'nearest' });
+    }
+  }
+
+  setFocus(focus) {
+    this.setState({
+      returnFromStaff: focus,
+    });
```
Comment (mikix) [inline src/components/EditCoursePage/CollapsibleCourseRun.jsx]: This method name is very genericly named, but the state it toggles is very specificly named. Maybe generic-ize the state name or specific-ize the method name?

Comment (Dillon-Dumesnil) [reply src/components/EditCoursePage/CollapsibleCourseRun.jsx]: +1

File: src/data/reducers/stafferInfo.test.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import stafferInfo from './stafferInfo';
 
 
 describe('stafferInfo reducer', () => {
-  let initalState;
+  let initialState;
```
Comment (Dillon-Dumesnil) [inline src/data/reducers/stafferInfo.test.js]: 🙏

File: src/components/EditCoursePage/CollapsibleCourseRun.jsx [hunk 3]
```diff
@@ -87,10 +87,54 @@ class CollapsibleCourseRun extends React.Component {
...
+  }
+
+  componentDidMount() {
+    const {
+      stafferInfo: { returnToEditCourse },
+      sourceInfo: { referringRun },
+      courseRun: { key },
+    } = this.props;
+
+    if (returnToEditCourse) {
```
[the remark this answers is not in the document]
Comment (ciduarte) [reply src/components/EditCoursePage/CollapsibleCourseRun.jsx]: Good catch!

**Pull request merged**